analyzers/nodejs/integration_test.go

```diff
@@ -130,76 +197,100 @@ func initializeProjects(testDir string) error {
 			}
 
 			// any key will work to prevent the "NEED KEY" error message
-			err = runfossa.Init(projectDir)
+			_, err = runfossa.Init(projectDir)
 			if err != nil {
 				println("failed to run fossa init on " + proj.Name)
 				println(err.Error())
 				panic(err)
 			}
 		}(project)
 	}
-
 	waitGroup.Wait()
 
 	return nil
 }
 
-var projects = []fixtures.Project{
-	fixtures.Project{
-		Name:   "puppeteer",
-		URL:    "https://github.com/GoogleChrome/puppeteer",
-		Commit: "b97bddf8e5750d20c6ba82392eebe2a3fd2dd218",
+type NodejsProject struct {
```

Comment:
I'm not convinced that it makes sense to separate `NodeJSProject` from `fixtures.Project`. Many other test fixtures require special arguments to analyse correctly.

Reply:
The `fixture.Project` is so that something can be `git clone`d. The extension of that (adding env) is unique to the environment. Its information that is not needed within the cloning context.

That is why I created a `NodejsProject` with `env` to extend the case for when `npm install` is being run.

Reply:
I think `Args` should be moved into the fixture regardless, because there are many non-NodeJS fixtures (e.g. Go) that require special args to analyse.

Re: `Env`:

From an ontological perspective, I agree.

From a developer experience, `NodejsProject` seems like a lot more boilerplate for very little added value. The two questions I'm considering here are:

- Will we ever use `fixture.Projects` in a way that having `Env` is confusing? (I don't think so.)
- Is it foreseeable that non-Nodejs projects will need to set `Env`? (Yes -- consider tests where we'd like to set e.g. `FOSSA_GRADLE_BINARY` for weird `gradlew` wrappers).

Reply:
I think that's a fair point. I can move `Env` and `Options` into the base and see it be reasonable. I'll take that approach